tests/files/locations/test_http.py

```diff
@@ -0,0 +1,60 @@
+import pathlib
+
+import pytest
+
+from astro.constants import FileLocation
+from astro.files.locations import location_factory
+from astro.files.locations.base import Location
+
+CWD = pathlib.Path(__file__).parent
+
+sample_filepaths_per_location = [
+    (FileLocation.HTTP, "http://domain/some-file"),
+    (FileLocation.HTTPS, "https://domain/some-file"),
+]
+sample_filepaths = [items[1] for items in sample_filepaths_per_location]
+sample_filepaths_ids = [items[0].value for items in sample_filepaths_per_location]
+
+
+def describe_get_location_type():
+    """test get_location_type()"""
```

Comment:
This docstring does not add any value! Either we need add more details or remove it entirely

Reply:
I know, but the deepsource check is failing if we don't add the docsting. I'll try to add more details in docstring which make sense.